File: service2.js
const express = require('express');
const { execSync } = require('child_process');

const app = express();

// Function to get system info
function getSystemInfo() {
    const ipAddress = execSync('hostname -I').toString().trim();
    const processes = execSync('ps -ax').toString();
    const diskSpace = execSync('df -h /').toString();
    const uptime = execSync('uptime -p').toString().trim();

    return {
        "IP Address": ipAddress,
        "Processes": processes,
        "Disk Space": diskSpace,
        "Uptime": uptime
    };
}

app.get('/', (req, res) => {
    const systemInfo = getSystemInfo();
    res.json(systemInfo);
});

app.listen(5000, () => {
    console.log('Service2 running on port 5000');
});

File: service2.py
from flask import Flask, jsonify
import subprocess

app = Flask(__name__)

# Function to get system info
def get_system_info():
    ip_address = subprocess.getoutput("hostname -I").strip()
    processes = subprocess.getoutput("ps -ax")
    disk_space = subprocess.getoutput("df -h /")
    uptime = subprocess.getoutput("uptime -p")
    return {
        "IP Address": ip_address,
        "Processes": processes,
        "Disk Space": disk_space,
        "Uptime": uptime
    }

@app.route('/', methods=['GET'])
def index():
    system_info = get_system_info()
    return jsonify(system_info)

if __name__ == '__main__':
    app.run(host='0.0.0.0', port=5000, debug=True)
